wfcGen.py

```diff
@@ -2,83 +2,79 @@
 
 from PIL import Image
 
+def loadCells(imageLink, tileSize=3, cropShift=False):
+    cells = {}
+    cropped = []
+    startImage = Image.open(imageLink)
+    for ins in range(4):
+        print(ins/4)
+        
+        cropped.append([])

```

Comment:
As I can see, on every iteration you insert an empty array here, and then overwrite it later by writing something to `cropped[ins]`. Does it make sense to remove the `append` here and, instead, do `cropped.append` in the end, when you have data to append?